Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py

```diff
@@ -75,20 +75,44 @@ def __init__(self, tenant_id: str = '',
         self.auth_type = SELF_DEPLOYED_AUTH_TYPE if self_deployed else OPROXY_AUTH_TYPE
         self.verify = verify
 
-    def http_request(self, *args, **kwargs):
+    def http_request(self, headers=None, *args, **kwargs):
```

Comment:
you can't pass keyword argument before positional args
```suggestion
    def http_request(self, *args, headers=None, **kwargs):
```